### DISCUSSION TITLE
Traffic Mod Save issue

### DISCUSSION THREAD

**Discussion by ozcan181** [krzychu124/Traffic#1] [Q&A] [answered]
It great  and wonderful mod but I am having save issue I mean after I modified the junction lanes setting all back to vanilla if I restart the game . Any  suggestion. Btw no error or incompatible mods in the game.

**Comment by krzychu124** [comment]
Can you provide the savegame and logs? I think you can share the savegame on PDX Mods (click "more options" on Load screen and then "Share savegame")

Game logs can be found in: `%userprofile%\AppData\LocalLow\Colossal Order\Cities Skylines II`, you can simply copy & paste the path into the folder address bar to navigate, find `Player.log` and from `\Logs\Traffic.Mod.log`

**Reply by ozcan181** [reply]
Hi thanks for your reply ,
please see the attached 

[Player.log](https://github.com/krzychu124/Traffic/files/15228404/Player.log)
And I sahe the game under pasting the this discussion link Asset ID 80335

**Reply by krzychu124** [reply, marked as answer]
it looks like mod was not loaded or not even subscribed when you tried loading the savegame. Mod data was stored correctly but because of missing mod, nothing has been loaded. If mod is loaded you can see its options screen (under _Traffic_) when you enter game options.

**Comment by ozcan181** [comment]
Thanks alot

I think it works now

________________________________
